client/setec/fileclient.go

```diff
@@ -6,12 +6,17 @@ package setec
 import (
 	"context"
 	"encoding/json"
+	"errors"
 	"fmt"
 	"os"
 
 	"github.com/tailscale/setec/types/api"
 )
 
+var (
+	ErrNotImplemented = errors.New("not implemented")
+)
```

Comment:
I don't think we need to export this (yet)
```suggestion
var errNotImplemented = errors.New("not implemented")
```

But if we do want to export it, we should document it.